index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+        <section class="main-data flex">
+            <div class="inputs flex">
+                <input class="title-input" type="text" min="5" max="20" placeholder="Title">
+                <input class="author-input" type="text" min="5" max="20" placeholder="Author">
+                <button class="add-btn" type="button">Add</button>
```

Comment:
- [ ] The styling has still work to do in order to match the given wireframe, as was mentioned by the previous reviewer
![image](https://user-images.githubusercontent.com/62705947/222192622-3c25b62c-7bc2-49f9-858b-4e792a3b082d.png)
Wireframe:
![image](https://user-images.githubusercontent.com/62705947/222192745-27a47e85-e5e9-4e7f-897b-bd1044a34296.png)